spec/support/jasmine.json

```diff
@@ -3,9 +3,5 @@
   "spec_files": [
     "**/*[sS]pec.js"
   ],
-  "helpers": [
-    "helpers/**/*.js"
-  ],
-  "stopSpecOnExpectationFailure": false,
-  "random": true
+  "random": false
```

Comment:
Is the change to `false` intended?

Reply:
Yes, it was intended.

I copied over the value that was used from other repos. For example, `cordova-common`, `cordova-lib` and `cordova-fetch` had this attribute set to `false`.

For consistency, this is why I changed it. Additionally, this repo only has one test/spec so there is no randomness order to run. It will always run the same test same order.

Reply:
My gut feeling says: keep random `true` and rather fix the other repos to use randomness in tests. IMO there is no reason to run tests without randomness.

Reply:
Agreed. The fact aside that this repo only has one dummy test, we should strive to enable random tests in all repos. Since random is the default we could delete this config altogether in this repo